### PR TITLE
Helm chart for Find+Community+View

### PR DESCRIPTION
**Pull request by christopher-rayson-mf**
Also a chart to stand up single Content instance as query & index service.

### CODE REVIEW

File: helm/idol-find/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+version: "0.1.0"
+appVersion: "23.4.0"
+description: Adds ``Find`` UI and supporting servers (Community, View)
+dependencies:
+- name: idol-library
+  version: "0.1.0"
+  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
+- name: idol-licenseserver
+  version: "0.1.0"
+  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
```
Comment (daniel-lau-mf) [inline helm/idol-find/Chart.yaml:22]:
```suggestion
  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
  condition: idol-licenseserver.enabled
```

File: helm/idol-find/templates/ingress-aci.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  annotations:
+{{- if eq .Values.ingressType "nginx" }}
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/proxy-body-size: {{ .Values.ingressProxyBodySize }}
+{{- else if eq .Values.ingressType "haproxy" }}
+    haproxy.router.openshift.io/rewrite-target: / 
+    haproxy.router.openshift.io/proxy-body-size: {{ .Values.ingressProxyBodySize }}
+{{- end }}
+spec:
+  rules:
```
Comment (daniel-lau-mf) [inline helm/idol-find/templates/ingress-aci.yaml:29]: I needed this to get ingress working for me
```suggestion
spec:
  ingressClassName: nginx
  rules:
```

File: helm/idol-find/templates/_helpers.tpl [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# END COPYRIGHT NOTICE
+
+{{/* Generate idol-find deployment liveness probe timeouts */}}
+{{- define "idolfind.deployment.standardLivenessProbe" }}
+          initialDelaySeconds: {{ .initialDelaySeconds | default 8 | int }}
+          timeoutSeconds: {{ .timeoutSeconds | default 3 | int }}
+          periodSeconds: {{ .periodSeconds | default 10 | int }}
+          failureThreshold: {{ .failureThreshold | default 3 | int }}
+{{- end -}}
+
```
Comment (daniel-lau-mf) [inline helm/idol-find/templates/_helpers.tpl:19]: We could probably put this in idol-library

**Comment by daniel-lau-mf** [comment]
Mostly I think this looks fine.
Could do with simple README.md in each chart.
idol-find doesn't have a values.schema.json

File: helm/idol-find/templates/ingress-aci.yaml (hunk 2 above)
Comment (daniel-lau-mf) [reply helm/idol-find/templates/ingress-aci.yaml:29]: With default ingress class this goes away, though think we should maybe add this as an optional chart value.

File: helm/idol-find/Chart.yaml (hunk 1 above)
Comment (christopher-rayson-mf) [reply helm/idol-find/Chart.yaml:23]: 658f335

File: helm/idol-find/templates/ingress-aci.yaml (hunk 2 above)
Comment (christopher-rayson-mf) [reply helm/idol-find/templates/ingress-aci.yaml]: e305824

File: helm/idol-find/templates/_helpers.tpl (hunk 3 above)
Comment (christopher-rayson-mf) [reply helm/idol-find/templates/_helpers.tpl]: 46670e9

**Comment by christopher-rayson-mf** [comment]
> Could do with simple README.md in each chart.

97b872b

**Comment by christopher-rayson-mf** [comment]
> idol-find doesn't have a values.schema.json

b7b1107

**Review by daniel-lau-mf — COMMENTED** [review]
Think this looks fine - need to generate the release files and update the index.yaml

**Review by daniel-lau-mf — APPROVED** [review]

**Pull request merged**